src/router/routes.ts

```diff
@@ -59,16 +61,32 @@ export const routes = [
       requiresAuth: true
     }
   },
-  //
-  // TODO: handle steps differently because they
-  //       differ depending on company type (BC, CP)
-  //
+  {
+    path: '/step-3',
+    name: 'Step-3',
+    component: Step3Container,
+    meta: {
+      step: -1,
+      label: 'Add People and Roles',
+      requiresAuth: true
+    }
+  },
+  {
+    path: '/step-4',
+    name: 'Step-4',
+    component: Step4Container,
+    meta: {
+      step: -1,
+      label: 'Add People and Roles',
+      requiresAuth: true
+    }
+  },
```

Comment:
Curious about the routes, not a question we need to answer now but we have to consider that steps 3 & 4 will be different based off on Entity type. 

It appears you've done that by establishing the steps 3 & 4 framework to be a generic container. I guess the labels just threw me off at first :)

Reply:
Step 3 and step 4 container (view) will be the same. Which component is rendered inside that will depend on the entity type. That way we dont have to worry abt handling all this in routes or steps